dali/pipeline/operator/op_schema.h

```diff
@@ -641,6 +655,34 @@ class DLL_PUBLIC OpSchema {
     return deprecated_in_favor_of_;
   }
 
+  DLL_PUBLIC inline bool IsDeprecatedArg(const std::string& arg_name) const  {
+    return deprecated_arguments_.find(arg_name) != deprecated_arguments_.end();
+  }
+
+  DLL_PUBLIC inline std::string DeprecatedArgMsg(const std::string& arg_name) const  {
+    auto it = deprecated_arguments_.find(arg_name);
+    DALI_ENFORCE(it != deprecated_arguments_.end(),
+      make_string("Argument \"", arg_name, "\" is not marked as a deprecated argument"));
+    if (!it->second.deprecated_str.empty())
+      return it->second.deprecated_str;
+    return make_string("Argument \"", arg_name, "\" for operator \"", name_,
+                       "\" is now deprecated.");
```

Comment:
I'm not sure how I feel about the default here, but at least it fixes a bug with warning about deprecation of `output_dtype` in an op that didn't use it.

Maybe mention it in the doc for AddDeprecatedArg?